src/NzbDrone.Core/Notifications/Ntfy/NtfyProxy.cs

```diff
@@ -7,6 +7,7 @@
 using NLog;
 using NzbDrone.Common.Extensions;
 using NzbDrone.Common.Http;
+using NzbDrone.Core.Download.Clients.FreeboxDownload;
```

Comment:
isn't it strange to do use a different download client?

Reply:
Not sure how this got pulled in, but removed.